integration_tests/tests/in_cluster.rs

```diff
@@ -47,103 +48,169 @@ impl InClusterJanusPair {
-                Ok(helper_api_namespace),
+                Ok(helper_aggregator_api_auth_token),
+                Ok(divviup_api_namespace),
             ) => (
                 kubeconfig_path,
                 kubectl_context_name,
                 leader_namespace,
-                leader_api_namespace,
+                leader_aggregator_api_auth_token,
                 helper_namespace,
-                helper_api_namespace,
+                helper_aggregator_api_auth_token,
+                divviup_api_namespace,
             ),
             _ => panic!("missing or invalid environment variables"),
         };
 
         let cluster = Cluster::new(&kubeconfig_path, &kubectl_context_name);
 
+        let (collector_private_key, task_builder, _) = test_task_builders(vdaf, query_type);
+        let task = task_builder.with_min_batch_size(100).build();
+
         // From outside the cluster, the aggregators are reached at a dynamically allocated port on
         // localhost. When the aggregators talk to each other, they do so in the cluster's network,
         // so they need the in-cluster DNS name of the other aggregator, and they can use well-known
         // service port numbers. The leader uses its view of its own endpoint URL to construct
         // collection job URIs, so we will only patch each aggregator's view of its peer's endpoint.
-        let leader_endpoints = {
-            let mut endpoints = leader_task.aggregator_endpoints().to_vec();
-            endpoints[1] = Self::in_cluster_aggregator_url(&helper_namespace);
-            endpoints
+
+        let divviup_api_port_forward = cluster
+            .forward_port(&divviup_api_namespace, "divviup-api", 80)
+            .await;
+
+        // Create an account first. (We should be implicitly logged in as a testing user already,
+        // assuming divviup-api was built with the integration-testing feature)
+        let account = divviup_api_create_account(&divviup_api_port_forward).await;
+
+        // Enumerate the paired aggregators available to our account to see whether we need to pair
+        // the leader (it may have been paired by a previous test run).
+        let paired_aggregators =
+            divviup_api_list_aggregators(&divviup_api_port_forward, &account).await;
+
+        let leader_aggregator_id = match paired_aggregators
+            .iter()
+            .find(|paired| paired.dap_url == Self::in_cluster_aggregator_dap_url(&leader_namespace))
+        {
+            Some(paired) => paired.id.clone(),
+            None => {
+                let leader_aggregator_pair_req = NewAggregatorRequest {
+                    role: "Leader".to_string(),
+                    name: "leader".to_string(),
+                    api_url: Self::in_cluster_aggregator_api_url(&leader_namespace).to_string(),
+                    dap_url: Self::in_cluster_aggregator_dap_url(&leader_namespace).to_string(),
+                    bearer_token: leader_aggregator_api_auth_token,
```

Comment:
This request needs to go to `POST /aggregators` to pair it as a global aggregator and not bound to `account`. Also, we need to add `is_first_party: true` to the request. Finally, there's no harm to registering the same aggregator multiple times as a global, first-party aggregator, so we shouldn't bother checking if it's already paired.